src/components/TodoListItem.js

```diff
@@ -6,12 +6,11 @@ function TodoListItem({ item, onRemoveTodo }) {
   return (
     <>
       <li className={styles.ListItem}>
-        {" "}
-        {item.fields.Title}{" "}
+        {item.fields?.Title ? item.fields.Title : item.title}
         <button
           className={styles.button}
           type="button"
-          onClick={() => onRemoveTodo(item)}
+          onClick={() => onRemoveTodo(item.id)}
```

Comment:
Try to think of a way to show a loading or removing status while removing an item so the user knows the app is actually doing something you remove. You probably do something similar for adding an item :)